<launch>
	<arg name="input_path_topic" default="/move_base_simple/path_raw" />
	<arg name="output_path_topic" default="/move_base_simple/path" />

	<node name="optimal_path_planner" pkg="boustrophedon_optimal_path_planner" type="optimal_path_planner.py" output="screen">
    <remap from="/move_base_simple/path_raw" to="$(arg input_path_topic)"/>
		<remap from="/move_base_simple/path" to="$(arg output_path_topic)"/>
  </node>
</launch>
